src/helm/azure-pipelines-agent/templates/NOTES.txt

```diff
@@ -1,4 +1,6 @@
+{{- if (.Values.secret.create) -}}
```

Comment:
`organizationURL` won't be always passed into the free field `extraManifests`. This can be misleading.

I propose to keep the note, but update it to add a warning to the user.

Reply:
My idea here was to pass the whole secret (or sealed secret) as an `extraManifest` and thus not require the `organizationURL`, but I agree that it's not clear the the value is still required.

Does this sounds reasonable?

```suggestion
{{- if (.Values.secret.create) -}}
Check your Azure DevOps portal at to manage the Azure Pipelines Agent ({{ .Values.pipelines.organizationURL | quote | required "A value for .Values.pipelines.organizationURL is required" }}/_settings/agentpools).
{{- else -}}
Secret creation disabled, remember to create/update the secret {{ include "azure-pipelines-agent.secretName" . | quote }} and that the fields personalAccessToken and organizationURL are required
{{- end -}}
```